### DISCUSSION TITLE
Localizing wix bootstrapper and wix msi

### DISCUSSION THREAD

**Discussion by CharithaKatamReddy** [wixtoolset/issues#7678] [Questions]
I have a project which has Wix msi, wix bootstapper and c# setupUX project.

C# setupUX is used for UI purpose which gives binaries(dll), We added ths dll as payload in bootsrapperproject

Wix msi project generates msi which we package in bootstapper project.

Now we have started working on localization of complete setup, we were able to localize UI(setupUX project) and include them in bootstrap project.

And we were able to localize wix MSi as well separately but failed to load them in bootstrap project

1. Creating different MSI's for each language works fine when i package the perticular languge MSI to the bootstrap project
2. But, We have generated sing msi for multiple languages using troch.exe and creating mst's, so that msi works based on os language
3. Tested that single msi in different languages which works fine
4. but failed in loading the correct language msi based on os language when packed in bootsrapper project

Also failed in finding a way to localize the strigs coming from bootstapper project.

Please let me know if there is any solution for this or any perticular way we need to follow to get this done.

Hope i would get some help..

Thanks in advance!!!!

localization , msilocalization, wixinstaller, installer, wix, burn, bootstrapper, visual-studio, c#localization, msi, mst, transforms

**Comment by nirbar** [comment]
Localized MSI packages have different ProductCodes, which burn doesn't currently support
I made some work in my modified WiX3 (https://www.nuget.org/packages/panelsw.custom.wix/) to support MSI instances which work a similar, however not exactly, like localized packages

You may contact me if you want me to add localized MSI packages option to a customized WiX4
I'm sure FireGiant will do the same if you ask them

**Comment by CharithaKatamReddy** [comment]
Hi Nir Bar,

Thanks for the reply.. Would like to discuss more on this

How shall we connect .

Thanks
Charitha.

**Comment by nirbar** [comment]
@CharithaKatamReddy 
https://il.linkedin.com/in/nir-bar-530b39

**Reply by CharithaKatamReddy** [reply]
Hi nirbar,

couldn't message you in linkdin, Can you share your email ID

**Reply by nirbar** [reply]
@CharithaKatamReddy you can send connect request in linkedin
Alternatively use "contact" in my website www.panel-sw.com
